### PR TITLE
explorer: add multi-file import for Windows

### PR DESCRIPTION
**Pull request by Ajnasz**
Intorduce the ability to import multiple files at once on Windows.

The selected file paths are split by null characters and the directory
is appended to each filename. If any file fails to open, all opened
files are closed and the error is returned.

### CODE REVIEW

File: explorer/explorer_windows.go [hunk 1]
```diff
@@ -118,8 +120,61 @@ func (e *Explorer) importFile(extensions ...string) (io.ReadCloser, error) {
...
+	}
+
+	// Split the pathUTF16 by null characters
+	paths := make([]string, 0)
+	currentPath := make([]uint16, 0)
+	for _, char := range pathUTF16 {
+		if char == 0 {
+			if len(currentPath) > 0 {
+				paths = append(paths, windows.UTF16ToString(currentPath))
+				currentPath = make([]uint16, 0)
```
Comment (inkeliz) [inline explorer/explorer_windows.go:145]: I think it's better to use `currentPath = currentPath[:0]` so you re-use the same buffer and (in the best case) avoids an unnecessary allocation.

**Comment by Ajnasz** [comment]
@inkeliz, @whereswaldon 
Thank you for the comments. Added the change you suggested and also signed off statement.

**Review by whereswaldon — APPROVED** [review]
Looks good and works well. Thank you very much!

**Comment by whereswaldon** [comment]
Merged!
To git.sr.ht:~whereswaldon/gio-x
   a77f90e..3152c15  main -> main

**Pull request closed without merging**